PROPOSAL_TEMPLATE.sh

```diff
@@ -30,11 +30,11 @@ $(bash -xc "git log --format='%C(auto) %h %s' $(git rev-parse --short tags/prod)
 
 ## Wasm Verification
 
-To build the wasm module yourself and verify its hash, run the following commands from the root of the nns-dapp repo:
+To build the wasm module yourself and verify its hash, run the following commands from the root of the [nns-dapp repo](https://github.com/dfinity/nns-dapp):
 
 \`\`\`
 git fetch  # to ensure you have the latest changes.
-git checkout \`$(git rev-parse tags/release-candidate)\`
+git checkout $(git rev-parse tags/release-candidate)
```

Comment:
```suggestion
git checkout "$(git rev-parse tags/release-candidate)"
```
This makes the command safer in case git rev-parse returns an unexpected result.